doc/man/8/mount.ceph.rst

```diff
@@ -43,7 +43,8 @@ the secret from it.
 
 A sub-directory of the file system can be mounted by specifying the (absolute)
 path to the sub-directory right after ":" after the socket in the device part
-of the mount command.
+of the mount command. If both the '/' and sub-directory are omitted, the '/'
+will be used in kernel space.
```

Comment:
So basically you want to allow a device string of ":" rather than ":/" ? I don't get how that adds any tangible benefit.

Reply:
Maybe as Patrick mentioned we could omit them all if this change makes sense.

Reply:
No, you can't. Mount helpers receive a specific set of arguments from /bin/mount. If you drop the "device" portion, then that will break things...unless you intend to somehow make the device portion optional.

Again though, I ask......what's the point of this change? Having to give ":/" as a device string doesn't seem overly burdensome.

Reply:
Yeah, actually this change makes not much sense. Just for one case it may benefit the code coverage.

Reply:
Let's drop this PR then.
Thanks.